RemoteDesktopToVM.md

```diff
@@ -17,14 +17,21 @@ The Remmina control window will open on your desktop and you'll see a new icon f
 1. Click on "New" in the Remmina menu panel and enter the following:
     * Name: `CS-RDSH-02`
     * Group: _leave blank_
+    * Protocol: _make sure this is set to_ `RDP - Remote Desktop Protocol`
```

Comment:
You can probably shorten this to just `RDP - Remote Desktop Protocol` right? It's clearly instructions, so you don't really need  "make sure this is set to"

Reply:
yeah, that's better